src/kudu/consensus/raft_consensus.cc

```diff
@@ -1988,6 +2012,7 @@ Status RaftConsensus::BulkChangeConfig(const BulkChangeConfigRequestPB& req,
           break;
 
         case MODIFY_PEER: {
+          LOG(INFO) << "modifying peer" << peer.ShortDebugString();
```

Comment:
added for debugging?

Reply:
yeah, if this gets annoying we can rip it out later